src/expiring_hash_map/mod.rs

```diff
@@ -0,0 +1,200 @@
+//! Expiring Hash Map and related types. See [`ExpiringHashMap`].
+#![warn(missing_docs)]
+
+use futures::stream::StreamExt;
+use std::borrow::Borrow;
+use std::collections::HashMap;
+use std::fmt;
+use std::hash::Hash;
+use std::time::{Duration, Instant};
+use tokio02::time::{delay_queue, DelayQueue, Error};
+
+#[cfg(test)]
+mod tests;
```

Comment:
not-minor nit: the file isn't big enough to split these can we just have the tests and the map in the same file?

Reply:
I'll change it since you asked. However, I personally think this style of keeping tests mod in an adjacent file is easier to maintain.
I don't think I should have a strong opinion on how to organize files yet, since I'm still relatively new in this project, and haven't touched/seen the majority of the code. But these minor details usually matter to me. On this particular topic, I'll just say I like how I did it more. :smile: